### DISCUSSION TITLE
Can Ctrl Left Click from .tex file perform SyncTeX

### DISCUSSION THREAD

**Discussion by TryerGit** [James-Yu/LaTeX-Workshop#3653] [Q&A] [answered]
Hello,

I utilize the `Ctrl LeftClick` operation from browser pdf view back to .tex file. At present, `CtrlAltJ` takes one from .tex cursor position to pdf viewer. Is it possible to setup so as to have `Ctrl LeftClick` take one from the .tex file to the pdf file?

That way, the same operations takes one back and forth between .tex and .pdf file.

I checked whether the left click operation is captured as a keyboard movement but within VSCode, it does not seem to be captured.

I just wanted to check that this is a VSCode limitation for which there is no workaround in LW.

Thank you.

**Comment by James-Yu** [comment, marked as answer]
This is a vscode limitation and there is no workaround known to me.
